tests/e2e/specs/in-game/one-offs/2_twos-destroy-target-royal.spec.js

```diff
@@ -41,8 +40,53 @@ describe('Play TWOS', () => {
       });
     });
 
+    it('Resolves a two to destroy a facecard after being double countered', () => {
```

Comment:
@haaarsh619 I added this testcase to demonstrate the failure based on setting targetCard from the result instead of the oneOffTarget.

When players counter back and forth, the `targetCard` is set to the two they are countering, not the original target. But the `oneOffTarget` stays the same throughout the countering process: the original target, in this case the king.

This test was failing because when players counter and counter back, the targetCard was being treated as the previous counter, rather than the king. The fix was just setting `result.targetCard = result.oneOffTarget` in the beginning of `two()`.

It should be correct now; I'm prepping to merge